datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -2388,12 +2389,13 @@ mod tests {
     }
 
     fn test_plan() -> LogicalPlan {
+        let table_ref: Option<TableReference<'_>> = None;
```

Comment:
use `TableReference::none()` that you introduced?

Reply:
Excellent call. Done in 2c3b6d7e6